### PR TITLE
Online-vs-Offline-IDE

### PR DESCRIPTION
**Pull request by ScrumUser1**

### CODE REVIEW

**Comment by ScrumUser1** [comment]
current fix to IDE article

File: Online-vs-Offline-IDE.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+**\#Online Offline Editors/IDE**
```
Comment (raisedadead) [inline Online-vs-Offline-IDE.md]: Why do you have these `\` ? They are not letting the markdown to parse correctly.

Comment (raisedadead) [inline Online-vs-Offline-IDE.md]: Remove these slashes everywhere.

File: Online-vs-Offline-IDE.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ 
+
+\*An offline IDE will be more beneficial to a programmer rather than an online
+IDE due to the fact that large \*projects will slow down and that a software
+team will need a reasonable office internet service. If there is a major \*storm
+the connection will be affected causing many projects to be unfinished and
+incomplete. The only \*advantages of online IDE is that projects can be access
+everywhere; there are no setup and configurations. There \*is no hardware
+limitation since all information will be stores in cloud.
+
```
Comment (raisedadead) [inline Online-vs-Offline-IDE.md]: Remove all the extra new lines

**Comment by raisedadead** [comment]
@ScrumUser1 I have added more comments
Update your the branch for this PR on your fork. No need to close the Pull request.
It will get updated automatically.

File: Online-vs-Offline-IDE.md (hunk 1 above)
Comment (alayek) [inline Online-vs-Offline-IDE.md]: I have a feeling this document is generated by some WYSIWYG editor which generates Markdown for formatted text. Maybe the slashes were generated by that.

File: Online-vs-Offline-IDE.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+**\#Online Offline Editors/IDE**
+
+ 
+
+\*\*There are several offline IDE that programmers use in order to stay more
```
Comment (alayek) [inline Online-vs-Offline-IDE.md]: I am a programmer, and I have no idea what the phrase _Offline IDE_ means. Same with _Online IDE_.

Just so everyone reads this article, is clear what you are talking about, spare a few words on defining such non-standard terms. If that's too much trouble, at least link to some Wikipedia article or any other public text that discusses what these are.

File: Online-vs-Offline-IDE.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+productive to learn and be more productive at work\*\*
+
+ 
+
+1.Eclipse is a free editor that is both online and offline incorporates Java,
```
Comment (alayek) [inline Online-vs-Offline-IDE.md]: Link to Eclipse official web site.

File: Online-vs-Offline-IDE.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+C/C++, JavaScript, and PHP by IBM. Its \*biggest advantages are code completion
```
Comment (alayek) [inline Online-vs-Offline-IDE.md]: What do you mean by
> and PHP by IBM

?

Comment (alayek) [inline Online-vs-Offline-IDE.md]: > Its ~~\*biggest~~ biggest advantages are code completion

File: Online-vs-Offline-IDE.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+by tabbing method which saves a lot of time writing documentations. It \*has a
+build in syntax check to fix any mistype words while coding a project. All of
+features we've come to expect \*of an IDE are their code completion, templates,
```
Comment (alayek) [inline Online-vs-Offline-IDE.md]: Reading long, information dense paragraphs are difficult. Reader loses attention trying to break the wall of text.

As a thumb rule, don't write more than 2 sentences per para. 3 at most.

**Comment by Rafase282** [comment]
I was trying for fix all that and send a PR to his but it was too much. Extra lines, `\` needs to be removed. Bold needs to be used on key names. Links as stated already, sub titles, proper markdown lists.

File: Online-vs-Offline-IDE.md (hunk 5 above)
Comment (ScrumUser1) [inline Online-vs-Offline-IDE.md]: okay i will try to remove the slashes and add some references.

**Pull request closed without merging**